2333/2/libs/top_bad_words_analyzer.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'handler'
+require 'terminal-table'
+require_relative './analyzer_printer.rb'
+require_relative './names_searcher.rb'
+
+# VersusBattle text analyzer
+class TopBadWordsAnalyzer
+  def initialize(rappers, take_value)
+    @rappers = rappers
+    @take_value = take_value.to_i
+  end
+
+  def top_bad_words
+    AnalyzerPrinter.print_top_bad_words(analyze_rappers)
```

Comment:
You use `#analyze_rappers` and it's kind of tells what to instead of describing what we want to get as a result. I suggest using method `#top_sorted_rappers` instead. And it may use `#sorted_rappers`.

This will make our code more readable. What name tells you more info, "analyze rappers" or "top sorted rappers"? When I see "analyze" I don't know what kind of analyzing is applied. I don't know what it returns. So I have to go deeper and read method `#analyze_rappers` instead. When I see "top sorted rappers" I know that it's limited amount of rapers sorted by some criteria. I don't know what criteria is used for sorting but at least I know it's sorted. I need to go deeper only if I'm curious what criteria was used for sorting.

And in fact you may describe the criteria in method name as well ;).